src/components/SearchUsersTextField.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import Button from "@material-ui/core/Button";
+import ButtonGroup from "@material-ui/core/ButtonGroup";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+import ClickAwayListener from "@material-ui/core/ClickAwayListener";
+import Fade from "@material-ui/core/Fade";
+import Paper from "@material-ui/core/Paper";
+import Popper from "@material-ui/core/Popper";
+import MenuItem from "@material-ui/core/MenuItem";
+import MenuList from "@material-ui/core/MenuList";
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import SearchIcon from '@material-ui/icons/Search';
+
+class SearchUsersTextField extends Component{
+
+    render(){
+        let options = this.props.options
+        let originalOptionsCount = options.length
+        if(!originalOptionsCount) options.push({username: 'no users found', id: 0})
+        return (
+            <div>
+                <TextField
+                    id="outlined-margin-dense"
+                    margin='dense'
+                    variant='outlined'
+                    placeholder="Search for Users"
+                    onChange={this.props.onChange}
+                    style = {{backgroundColor: 'whiteSmoke', width: this.props.width}}
+                    fullWidth
+                    InputProps={{
+                        startAdornment: (
+                            <InputAdornment position="start">
+                                <SearchIcon />
+                            </InputAdornment>
+                        ),
+                    }}
+                />
+                <Popper
```

Comment:
I have no idea why you'd rather use this instead of the autocomplete component we have before.....

Reply:
the autocomplete doesn't work if you add an icon to it form some reasons. Also it's not flexible, I couldn't use an outlined textfield with a specific size.